src/api/goods.ts

```diff
@@ -3,17 +3,23 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
-  return fetch(API_URL)
-    .then(response => response.json());
+export async function getAll(): Promise<Good[]> {
+  const response = await fetch(API_URL);
+
+  return response.json();
 }
 
-export const get5First = () => {
-  return getAll()
-    .then(goods => goods); // sort and get the first 5
+export const get5First = async () => {
+  const goods = await getAll();
+
+  return [...goods]
```

Comment:
Why you use spread syntax here? I fink you can sort goods if you write "return goods.sort"